models/src/main/scala/coop/rchain/models/block/StateHash.scala

```diff
@@ -0,0 +1,13 @@
+package coop.rchain.models.block
+
+import com.google.protobuf.ByteString
+import coop.rchain.crypto.codec.Base16
+
+object StateHash {
+  type StateHash = ByteString
+
+  val Length = 32
+  implicit class StateHashOps(bs: StateHash) {
+    def base16String: String = Base16.encode(bs.toByteArray)
```

Comment:
This extension is almost identical to [BlockHashOps](https://github.com/rchain/rchain/blob/003e2a8e/models/src/main/scala/coop/rchain/models/BlockHash.scala#L10).

We can have only one defined for `ByteString` which will work in both cases. It should be defined in _syntax_ style so it's easier to import.
I'm ok if this is done as part of refactoring case #3001.